src/engine/input/RunOnMultiAxisAttribute.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using System.Linq;
+
+public class RunOnMultiAxisAttribute : RunOnInputAttribute
```

Comment:
Add documentation on this. I tried reading this for like a minute but couldn't really determine what goes on here.